stellar_sdk/server.py

```diff
@@ -39,17 +39,20 @@ class Server(BaseServer):
         (ex. ``"https://horizon-testnet.stellar.org"`` for test network,
         ``"https://horizon-testnet.stellar.org"`` for public network)
     :param client: Http client used to send the request
+    :param custom_request_headers: Any headers that should be added when making HTTP requests to the
```

Comment:
Personally, I prefer not to modify this file. Modifying headers is not a common requirement, and not every client implements the function of modifying headers. If a user has such a need, then he should create a client instance himself, it's not difficult in my opinion.